rel/vars.config.in

```diff
@@ -46,8 +46,8 @@
 
 {sm_backend, "{mnesia, []}"}.
 {auth_method, "internal"}.
-{cyrsasl_external, "{cyrsasl_external, standard}"}.
-{ext_auth_script, "%%{extauth_program, \"/path/to/authentication/script\"}."}.
+{password_format, "{password_format, plain}"}.
```

Comment:
How about going one step further and putting `{password_format, plain}` to `rel/files/mongooseim.cfg`? Is there a need for this value to be templated?

Reply:
It could be if we are to add it differently for mim1 in the near future. Am I correct?

Reply:
True... I forgot about that...